.github/workflows/checkpoint_feedist.yml

```diff
@@ -14,7 +14,20 @@ jobs:
       matrix:
         python-version: [3.8]
     steps:
-      - uses: actions/checkout@v2
+      - id: df-py
+        uses: pozetroninc/github-action-get-latest-release@master
+        with:
+          owner: oceanprotocol
+          repo: df-py
+          excludes: prerelease, draft
+
+      - name: Set branch to checkout
+        id: set_branch
+        run: echo "BRANCH=${{ github.event_name == 'schedule' && steps.df-py.outputs.release || github.ref }}" >> $GITHUB_ENV
```

Comment:
I think it's possible to use this directly in checkout, without setting it separately. You will have ref: ${{ steps.df-py.outputs.release }}

Reply:
Ok then here is what I suggest:
- set a default branch to handle the null setting. I don't know how it behaves if you do not set the env, but specify the ref key
- also, I believe BRANCH is a bit generic, can you name it to something more specific? E.g. DF_PY_LATEST_BRANCH or something similar?